src/app/search/search-item.model.ts

```diff
@@ -0,0 +1,54 @@
+export interface SearchItem {
+    "kind": string,
```

Comment:
Please divide this huge model to smaller independent peace's

Reply:
I added 2 additional interfaces